qiskit/algorithms/evolvers/__init__.py

```diff
@@ -15,6 +15,7 @@
 from .evolution_result import EvolutionResult
 from .evolution_problem import EvolutionProblem
 
+
```

Comment:
I guess this file is changed just due to one extra blank line added. No biggie - I don't think its needed though right!